tests/test-parse.js

```diff
@@ -58,6 +58,9 @@ t.equal(parsed.SPACED_KEY, 'parsed', 'parses keys and values surrounded by space
 const payload = dotenv.parse(Buffer.from('BUFFER=true'))
 t.equal(payload.BUFFER, 'true', 'should parse a buffer into an object')
 
+const macPayload = dotenv.parse(Buffer.from('MAC=true\rEOL=true'))
+t.same(macPayload, { MAC: 'true', EOL: 'true' }, 'can parse mac (\\r) line endings')
```

Comment:
Regarding your error, you need to update `t.plan` on line 12 to be 25 instead of 24 because you're adding this test. 

It would also be good to test for `\r\n` (from the regex). A combination of line endings might also be good to test just to cover our bases (but it's an unlikely real-world scenario).

Reply:
Ah i see, missed that t.plan. I'll add tests for all line endings.